templates/no_info.html

```diff
@@ -1,29 +1,11 @@
-{% extends "bootstrap/base.html" %}
-{% block head %}
-  {{ super() }}
-  <link rel="icon" href="{{ url_for('static', filename='favicon.ico') }}" type="image/x-icon">
-{% endblock %}
+<!-- {% extends "base.html" %}
 {% block title %}EMPTY QUERY!!!{% endblock %}
 
 {% block content %}
 <div class='container'>
-<h2>{{ subject }}</h2>
+<h2>{{ subject|replace('\n', '<br>')|safe }}</h2>
```

Comment:
I suggest the following (although linefeeds are hopefully no longer in the `subject`:
```suggestion
<h2>{{ subject|replace('\n', ' ')|safe }}</h2>
```